src/RokuDeploy.ts

```diff
@@ -470,7 +470,7 @@ export class RokuDeploy {
     }
 
     private checkRequest(results) {
-        if (!results || !results.response || typeof results.body !== 'string') {
+        if (!results?.response || typeof results?.body !== 'string') {
```

Comment:
These optional chaining things cause issues with code coverage. Let's revert this and adjust any eslint rule that's complaining about it.

Reply:
done